frontend/views/containers/payments/PaymentRowReceived.vue

```diff
@@ -71,6 +72,10 @@ export default ({
     PaymentNotReceivedTooltip,
     PaymentRow
   },
+  mixins: [PaymentsMixin],
+  data: {
+    relativeTo: null
+  },
```

Comment:
Please check the Vue.js documentation for how to set `data` properly. [`data` cannot be an object](https://v2.vuejs.org/v2/guide/components#data-Must-Be-a-Function), it must be a function that returns an object.

Reply:
Hmm, good idea, my Vue.js is visibly a bit rusty...